packages/scene-composer/src/components/panels/scene-components/PlaneGeometryComponentEditor.tsx

```diff
@@ -72,6 +80,43 @@ export const PlaneGeometryComponentEditor: React.FC<IPlaneGeometryComponentEdito
     [internalHeight],
   );
 
+  const onTextureSelectClick = useCallback(() => {
+    if (showAssetBrowserCallback) {
+      showAssetBrowserCallback((s3BucketArn, contentLocation) => {
+        let textureUri: string;
+        if (s3BucketArn === null) {
+          // This should be used for local testing only
```

Comment:
nit-pick: Suggest updating the name to reflect the intent. ex: instead of `textureUri` make it `localTestTextureUri`.